lesson_06/conditionals/conditionals_app/src/main/java/com/codedifferently/lesson6/Lesson6.java

```diff
@@ -81,7 +117,17 @@ public static double addNumbers(double[] values) {
    * @return An array containing the first `n` fibonacci values.
    */
   public static int[] getFirstNFibonacciNumbers(int n) {
-    return null;
+    int[] fibNums = new int[n];
+    int num1 = 1;
+    int num2 = 1;
+    for (int i = 0; i < n; i++) {
+      fibNums[i] = num1;
```

Comment:
Would be more naturally expressed as `fibNums[i] = fibNums[i - 1] + fibNums[i - 2]`, then special case values of `n < 2`.

Reply:
Implemented more natural expression and special cases for values less than 2.